catalyst-gateway/bin/src/db/event/signed_docs/tests/mod.rs

```diff
@@ -0,0 +1,37 @@
+//! Integration tests of the `signed docs` queries
+
+use super::*;
+use crate::db::event::establish_connection;
+
+#[ignore = "An integration test which requires a running EventDB instance, disabled from `testunit` CI run"]
+#[tokio::test]
+async fn some_test() {
+    establish_connection();
+
+    let docs = [
+        (
+            uuid::Uuid::now_v7(),
+            uuid::Uuid::now_v7(),
+            uuid::Uuid::now_v7(),
```

Comment:
```suggestion
            uuid::Uuid::now_v4(),
```
Technically, this would be a UUIDv4, and while a v7 would almost certainly work for a test, better to check it as a v4 for completeness.